keyboards/ducky/one2sf/1967st/rules.mk

```diff
@@ -30,3 +30,6 @@ RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
 AUDIO_ENABLE = no           # Audio output
 DIP_SWITCH_ENABLE = yes
 
+RGB_MATRIX_ENABLE = yes
+
+SRC += rgb_matrix_drivers.c
```

Comment:
The enable should be moved to the info.json, and the driver config can be moved to 1967sh.c, so the `SRC` part isn't necessary.

Reply:
Thanks for the feedback. Should I move the other enable options into the info.json as well?

Reply:
Done!